src/core/components/dialogs/dialog/dialogRoot/dialogRoot.tsx

```diff
@@ -50,6 +51,16 @@ export interface IDialogRootProps extends ComponentPropsWithoutRef<'div'> {
     onPointerDownOutside?: (e: Event) => void;
 }
 
+const overlayAnimationVariants = {
+    closed: { opacity: 0 },
+    open: { opacity: 1 },
+};
+
+const contentAnimationVariants = {
+    closed: { opacity: 0, scale: 0.88, y: 100 },
+    open: { opacity: 1, scale: 1, y: 0 },
+};
```

Comment:
Should we move these two objects to a `dialogUtils` file under `/dialogs` to reuse the same animation settings for both dialogs? I don't think we'll have different animations for dialogs & alert-dialogs